portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,13 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Run!',
+        'Don\'t blink.',
+        'I\'m planning on dyeing my hair pink.',
+        'I have a cat named Gracie.',
+        'It\'s dangerous to go alone! Take this.',
+        'You want weapons? We\'re in a library! Books! The best weapons in the world!',
```

Comment:
Dr. Who? There's a David Tenant episode in a library, and this sounds like it's from that.

Reply:
Right TV show!